english/netflix/netflix.md

```diff
@@ -16,4 +16,4 @@ Watch the Passion of the Christ instead
 
 Until you realised you hate Mel Gibson
 
-So instead watch Breaking Bad for the 12th time.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.